test/math/kd_tree_test.cc

```diff
@@ -101,23 +100,121 @@ TEST_F(NodeCmpTest, Test) {
+  const auto fourth_level_node_7 = third_level_node_4->get_left();
+  const auto fourth_level_node_8 = third_level_node_4->get_right();
+
+  // Verify the regions.
+  // second level
+  EXPECT_EQ(second_level_node_1->get_region().min_corner(), second_level_region_1.min_corner());
+  EXPECT_EQ(second_level_node_1->get_region().max_corner(), second_level_region_1.max_corner());
+  EXPECT_EQ(second_level_node_2->get_region().min_corner(), second_level_region_2.min_corner());
+  EXPECT_EQ(second_level_node_2->get_region().max_corner(), second_level_region_2.max_corner());
+  // third level
+  EXPECT_EQ(third_level_node_1->get_region().min_corner(), third_level_region_1.min_corner());
+  EXPECT_EQ(third_level_node_1->get_region().max_corner(), third_level_region_1.max_corner());
+  EXPECT_EQ(third_level_node_2->get_region().min_corner(), third_level_region_2.min_corner());
+  EXPECT_EQ(third_level_node_2->get_region().max_corner(), third_level_region_2.max_corner());
+  EXPECT_EQ(third_level_node_3->get_region().min_corner(), third_level_region_3.min_corner());
+  EXPECT_EQ(third_level_node_3->get_region().max_corner(), third_level_region_3.max_corner());
+  EXPECT_EQ(third_level_node_4->get_region().min_corner(), third_level_region_4.min_corner());
+  EXPECT_EQ(third_level_node_4->get_region().max_corner(), third_level_region_4.max_corner());
+  // fourth level
+  EXPECT_EQ(fourth_level_node_1->get_region().min_corner(), fourth_level_region_1.min_corner());
+  EXPECT_EQ(fourth_level_node_1->get_region().max_corner(), fourth_level_region_1.max_corner());
+  EXPECT_EQ(fourth_level_node_2->get_region().min_corner(), fourth_level_region_2.min_corner());
+  EXPECT_EQ(fourth_level_node_2->get_region().max_corner(), fourth_level_region_2.max_corner());
+  EXPECT_EQ(fourth_level_node_3->get_region().min_corner(), fourth_level_region_3.min_corner());
+  EXPECT_EQ(fourth_level_node_3->get_region().max_corner(), fourth_level_region_3.max_corner());
+  EXPECT_EQ(fourth_level_node_4->get_region().min_corner(), fourth_level_region_4.min_corner());
+  EXPECT_EQ(fourth_level_node_4->get_region().max_corner(), fourth_level_region_4.max_corner());
+  EXPECT_EQ(fourth_level_node_5->get_region().min_corner(), fourth_level_region_5.min_corner());
+  EXPECT_EQ(fourth_level_node_5->get_region().max_corner(), fourth_level_region_5.max_corner());
+  EXPECT_EQ(fourth_level_node_6->get_region().min_corner(), fourth_level_region_6.min_corner());
+  EXPECT_EQ(fourth_level_node_6->get_region().max_corner(), fourth_level_region_6.max_corner());
+  EXPECT_EQ(fourth_level_node_7->get_region().min_corner(), fourth_level_region_7.min_corner());
+  EXPECT_EQ(fourth_level_node_7->get_region().max_corner(), fourth_level_region_7.max_corner());
+  EXPECT_EQ(fourth_level_node_8->get_region().min_corner(), fourth_level_region_8.min_corner());
+  EXPECT_EQ(fourth_level_node_8->get_region().max_corner(), fourth_level_region_8.max_corner());
```

Comment:
Similar comment here. I think we need to set the exact value and validate that points are also equal between regions. But providing the value makes the test more explicit and accurate.

Reply:
I am validating the regions with the values defined at: https://github.com/maliput/maliput/pull/520/files#diff-a95ace39501f3ade8677b105d0ee1922ca1255286ccdecaa896fd758ffe5ccafR137-R152